guides/pricing_and_taxation.md

```diff
@@ -0,0 +1,127 @@
+# Pricing and Taxation
+
+Taxation and prices are two intertwined topics in Solidus, mainly because a large number of jurisdictions in the world use VAT-style taxation, which influences the price at which an item is sold.
+
+## Taxation
+
+Solidus represents taxes for an order's items by using `Spree::TaxCategory` and `Spree::TaxRate`.
+
+Tax rates are associated with a tax category and a zone. If an order's `tax_address` (usually the shipping address) falls within a zone with tax rates, those rates will apply to all line items and shipments with respectively matching tax categories.
+
+When a tax rate gets applied to a line item, two things happen: A tax adjustment is created on the line item, and the line item's `additional_tax_total` and `included_tax_total` are changed to reflect the newly created adjustments. The amounts are then added to the order's `*_tax_total` fields.
```

Comment:
What's the difference between `additional_tax_total` and `included_tax_total`? Perhaps link down to the lower section? Or does the more conceptual, introductory section "Sales tax vs. Value-Added Tax" belong higher up in this document?